test/blackbox/common/mesh_event_handler.h

```diff
@@ -90,6 +90,20 @@ struct cond_flag {
 	bool flag;
 };
 
+typedef struct {
+	mesh_event_t  currentEv;
+	mesh_event_t  nextEv;
+} meshlink_node_status_lookup_t;
```

Comment:
Avoid using names that start with `meshlink_` outside of the main MeshLink code, it makes it harder to see if something is part of the main code or the test suite. I would just drop the `meshlink_` prefix for thse two structs.

Reply:
From the code it looks like you always do something like:
```
meshlink_node_status_lookup_t foo {
    EVENT1, EVENT2,
    EVENT2, EVENT3,
    EVENT3, EVENT4, ...
};
```
So you are always repeating the previous `meshlink_node_status_lookup_t`'s `nextEv` as the next one's `currentEv`. That seems redundant to me. Why not just use a simple array of `mesh_event_t`'s? So instead:
```
meshlink_node_status_lookup_t *lookup;
```
Use:
```
const mesh_event_t *expected_events;
```

Reply:
I have made the changes